evaluation_script/main.py

```diff
@@ -136,7 +124,7 @@ def calculate_base_metrics(infile, truth):
                 global_metrics.add_fp(1)
                 query_level_metrics[query_id].add_fp(1)
 
-    return global_metrics
+    return (global_metrics, query_level_metrics)
 
 def populate_ground_truth(infile):
```

Comment:
Update the docstring to describe the return values. Currently it describes a single dictionary.

Reply:
I think you missed this one. I'm referring to line 131, which says:
```
A dictionary of every query-document pair having a judgement is returned. 
```
This is no longer correct. Three values are returned, representing (a) dictionary of query-document pairs having judgements; (b) set identifying doc-ids having judgements; (c) dictionary mapping query-ids to a Metrics instance that will be used for per-query metrics calculations.

Also, the lines at 137-138 are no longer correct. These talk about initializing the global objects, which are now being returned.

Reply:
addressed in latest revision